Dockerfile

```diff
@@ -0,0 +1,49 @@
+FROM python:2.7.8-wheezy
+MAINTAINER James Swanick "swanijam@gmail.com"
+
+RUN curl http://python-distribute.org/distribute_setup.py | `which python`
+RUN curl https://raw.github.com/pypa/pip/master/contrib/get-pip.py | `which python`
+
+RUN apt-get update -qq
+RUN apt-get install unzip -y
+RUN apt-get install fish -y
+RUN apt-get install mktemp
+RUN apt-get install -y build-essential\
+    git-core\
+    libldap2-dev\
+    libpq-dev\
+    libsasl2-dev\
+    libssl-dev\
+    libxml2-dev\
+    libxslt1-dev\
+    libffi-dev\
+    python-dev\
+    python-setuptools\
+    zlib1g-dev\
+    postgresql-client
+
+RUN pip install harvest
+RUN harvest init Project --no-input
+
+ADD requirements.txt ./Project-env/
+
+WORKDIR Project-env
+
+RUN ["/bin/bash", "-c", "source bin/activate"]
+RUN pip install -r ./requirements.txt
+
+WORKDIR ./Project
+
+RUN touch tmpfile
+RUN head -n 16 ./Project/conf/global_settings.py > tmpfile
+RUN echo "'djredcap'," >> tmpfile
+RUN tail -n +17 ./Project/conf/global_settings.py >> tmpfile
+RUN mv tmpfile ./Project/conf/global_settings.py
```

Comment:
Remove all this temp stuff and add an extra file in the repository called `local_settings.py` that will be added to the `project/conf`directory containing the following:

```python
from global_settings import INSTALLED_APPS

INSTALLED_APPS += (
    'djredcap',
)

MODELTREES = {
    'default': {
        'model': 'project.Record',
    }
}
```